test/api/buffer/receivers/tasks/002-get-unfinished-ok/expected-response.json

```diff
@@ -0,0 +1,12 @@
+{
+  "id": "2023-02-01T08:02:06.000Z_ghijkl",
+  "receiverId": "my-receiver",
+  "url": "https://%s/v1/receivers/my-receiver/tasks/receiver.create/2023-02-01T08:02:06.000Z_ghijkl",
+  "type": "receiver.create",
+  "createdAt": "2023-02-01T08:02:06.000Z",
+  "finishedAt": "",
+  "isFinished": false,
+  "duration": 0,
+  "result": "receiver created",
+  "error": ""
```

Comment:
Tu vidim, ze error zostal string, nie je tu `null`.

Reply:
(v kode mas &error)

Reply:
To myslíš tady? https://github.com/keboola/keboola-as-code/blob/832d498138573ac730daac3a22595948b9d6ff16/internal/pkg/service/buffer/api/service/mapper/task.go#L27 A jak to mám udělat? Jakmile ten field v designu není required, tak ho Goa vygeneruje jako `*string`.

Reply:
Vygeneruje ho spravne, ale ty tam VZDY das pointer na string, takze vo vysledku je vzdy string.

```
var errPtr *string
if model.Error != "" {
  errPtr = &model.Error
}
```

Dava to takto zmysel?

Reply:
To iste je aj s `finishedAt`, ... alebo inak, potrebujes tam mat `nil`, nie pointer na prazdny string.

Reply:
To jako takhle? 
![image](https://user-images.githubusercontent.com/215660/218679910-59610436-05b3-4d3b-9bd0-827295afb6bf.png)

Ty dva fieldy ale budou v response chybět když jsou `null`.
![image](https://user-images.githubusercontent.com/215660/218680125-c90f4282-8dab-4bd7-a0ea-700ce8cfba44.png)

Reply:
Aha, tak to vypadá že `error: null` je v jsonu ekvivalentní tomu když chybí. Myslím že UIčku se to kdysi nelíbilo, ale třeba už to tak není. 🙂 Fixnuto: https://github.com/keboola/keboola-as-code/pull/1190/commits/954f17cf96f40762960c7a9df4367caa5740b548

Reply:
Goa by default nedava do result optional field ak je `null`.

Da sa to vynutit s:
```
Meta("struct:tag:json", "finishedAt") // remove "omitempty", include null in response
```

Ja v tom problem nevidim.

Ak chceme aby boli v response vsetky optional fields, tak by bolo lepsie na to napisat jednoduche `goa extension`, ktore to `omitempty` odstrani vzdy.

Reply:
> Myslím že UIčku se to kdysi nelíbilo, ale třeba už to tak není.

Neviem, ak to bude problem, kludne napisem tu goa extension, a pribudnu vsade `null` hodnoty.
V Templates API su tam tiez nejake optional fields, a asi to problem nebol, nepamatam.

Reply:
Asi bych to neřešil. Googlil jsem a zjevně je to json standard že field může chybět.